### DISCUSSION TITLE
[Vanilla Bug]/[New Feature] Cloaked units on water still have wakes

### DISCUSSION THREAD

**Discussion by MNOrac** [Vinifera-Developers/Vinifera#1528] [Feature Requests]
### Description:
A wake get drawn when a unit crosses a water tile, whether the unit itself is visible to the player or not. 

### Steps To Reproduce:
1. Make a cloaked vehicle amphibious, or an amphibious vehicle cloaked
2. Ask an enemy player in an multiplayer match to drive it over some water
3. Observe result

As-is, it's a very minor (probably acceptable) inconvenience for a cloaked Hover MLRS, but a serious issue for a submarine.

**Comment by Lylaktech** [comment]
If this could be made into an tag defined within the unit's code instead of being universally fixed I think it would be better to keep the option if a modder want to balance out their cloaked amphibious unit with wakes.

**Comment by CCHyper** [comment]
@MNOrac:
Great Item, I am going to upgrade this to also be a feature request as I feel it falls somewhere in between both.
